bootstrap/sql/mysql/create_tables.sql

```diff
@@ -14,15 +14,18 @@
 
 -- THE NAMES OF THE TABLE COLUMNS MUST MATCH THE NAMES OF THE CORRESPONDING CLASS MODEL FIELDS
 
+-- USE schema_registry;
+
 CREATE TABLE IF NOT EXISTS schema_metadata_info (
-  id            BIGINT AUTO_INCREMENT NOT NULL,
-  type          VARCHAR(256)          NOT NULL,
-  schemaGroup   VARCHAR(256)          NOT NULL,
-  name          VARCHAR(256)          NOT NULL,
-  compatibility VARCHAR(256)          NOT NULL,
-  description   TEXT,
-  evolve        BOOLEAN               NOT NULL,
-  timestamp     BIGINT                NOT NULL,
+  id              BIGINT AUTO_INCREMENT NOT NULL,
+  type            VARCHAR(256)          NOT NULL,
+  schemaGroup     VARCHAR(256)          NOT NULL,
+  name            VARCHAR(256)          NOT NULL,
+  compatibility   VARCHAR(256)          NOT NULL,
+  validationLevel VARCHAR(256)          NOT NULL, -- added in 0.3.1, table should be altered to add this column from earlier versions.
```

Comment:
@michaelandrepearce I did not have those changes as you already have them in other PR. You can merge your changes in PR and mention in the comments what versions work fine for this script in the comments.